YAML/YAML.sublime-syntax

```diff
@@ -171,9 +171,17 @@ variables:
   _type_int_other: ([-+]?)([1-9][0-9_]*(?::[0-5]?[0-9])+) # (base 60)
 
   # http://yaml.org/type/float.html
-  _type_float_decimal: ([-+]?)((?:[0-9][0-9_]*)?(\.)[0-9.]*(?:[eE][-+][0-9]+)?)
+  _type_float_decimal: |-
+    (?x)
+    ([-+]?) (
+      (?:
+        (?:[0-9][0-9_]*) (\.) [0-9_]*
+      | (\.) (?:[0-9][0-9_]*)
```

Comment:
```suggestion
      | (\.) (?:_*[0-9][0-9_]*)
```

Reply:
In that case we can simplify the pattern to

```suggestion
      | (\.) [0-9_]+
```